README.md

```diff
@@ -40,6 +40,19 @@ image:
 Additionally for the best user experience and optimized `sti` operation we suggest images
 to have `/bin/sh` and `tar` commands available.
 
+Pruning the contents of the source tree is possible if the user supplies a
+`.s2iignore` file in the root directory of the source repository.  Supply line by line the regular
+expression that best captures a set of files and directories you want removed.  You can also cite
```

Comment:
+1 @soltysh, I actually debated suggesting "filtering" instead of "pruning" when I was reviewing this, but I didn't want to get too picky :)